<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>warning</class>
 <widget class="QWidget" name="warning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warning</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lab_warning">
     <property name="text">
      <string>Folder exists!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btn_proceed">
      <property name="text">
       <string>PROCEED</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>CANCEL</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
